pkg/quay/quay.go

```diff
@@ -0,0 +1,89 @@
+package quay
```

Comment:
Is this package specific to quay, or would it work with another registry, too?

Reply:
Yes quay.io only.